tools/src/Packages.ts

```diff
@@ -379,7 +379,13 @@ export async function getListOfPackagesAsync(): Promise<Package[]> {
   if (!cachedPackages) {
     const paths = await glob('**/package.json', {
       cwd: PACKAGES_DIR,
-      ignore: ['**/example/**', '**/node_modules/**', '**/__tests__/**', '**/__mocks__/**'],
+      ignore: [
+        '**/example/**',
+        '**/node_modules/**',
+        '**/__tests__/**',
+        '**/__mocks__/**',
+        '**/__fixtures__/**',
```

Comment:
I had to add `__fixtures__` as there are some package.json in `jest-expo` that we don't want to publish